native/packages/react-native-bpk-component-button/readme.md

```diff
@@ -0,0 +1,34 @@
+# react-native-bpk-component-button
+
+> Backpack React Native button component.
+
+## Installation
+
+```sh
+npm install react-native-bpk-component-button --save-dev
+```
+
+## Usage
+
+```js
+import React, { Component } from 'react';
+import { View } from 'react-native';
+import BpkButton from 'react-native-bpk-component-button';
+
+export default class App extends Component {
+  render() {
+    return (
+      <View>
+        <BpkButton title="Book flight" onPress={() => {}} />
```

Comment:
I'm keen for these readme code examples to match the screenshot we have in the docs as much as possible. I.e. if you take the text component for example - it has a container with some padding etc (see https://github.com/Skyscanner/backpack/tree/master/native/packages/react-native-bpk-component-text). I guess this can be refined when you approach the docs for this component.